app/experimenter/experiments/serializers.py

```diff
@@ -45,8 +45,11 @@ def to_representation(self, data):
             for field in self.child.fields:
```

Comment:
How about up here doing something like

    initial_fields = set(self.child.fields) - set(["id"])

?

Reply:
👍 yes set differences.... why didn't i think of that 🤦‍♀

Reply:
😉